plugin-localization/src/main/java/com.mapbox.mapboxsdk.plugins.localization/MapLocale.java

```diff
@@ -0,0 +1,145 @@
+  static final LatLngBounds GERMANY_BBOX = new LatLngBounds.Builder()
+    .include(new LatLng(55.055637, 5.865639))
+    .include(new LatLng(47.275776, 15.039889)).build();
+  static final LatLngBounds KOREA_BBOX = new LatLngBounds.Builder()
+    .include(new LatLng(38.612446, 125.887108))
+    .include(new LatLng(33.190945, 129.584671)).build();
+  static final LatLngBounds JAPAN_BBOX = new LatLngBounds.Builder()
+    .include(new LatLng(45.52314, 122.93853))
+    .include(new LatLng(24.249472, 145.820892)).build();
+  static final LatLngBounds FRANCE_BBOX = new LatLngBounds.Builder()
+    .include(new LatLng(51.092804, -5.142222))
+    .include(new LatLng(41.371582, 9.561556)).build();
+  static final LatLngBounds ITALY_BBOX = new LatLngBounds.Builder()
+    .include(new LatLng(47.095196, 6.614889))
+    .include(new LatLng(36.652779, 18.513445)).build();
+  static final LatLngBounds PRC_BBOX = new LatLngBounds.Builder()
+    .include(new LatLng(53.56086, 73.557693))
+    .include(new LatLng(15.775416, 134.773911)).build();
+  static final LatLngBounds TAIWAN_BBOX = new LatLngBounds.Builder()
+    .include(new LatLng(25.29825, 119.534691))
+    .include(new LatLng(21.901806, 122.000443)).build();
+
+  /**
+   * Some countries already defined
+   */
+  public static final MapLocale FRANCE = new MapLocale(FRENCH, FRANCE_BBOX);
+  public static final MapLocale GERMANY = new MapLocale(GERMAN, GERMANY_BBOX);
+  public static final MapLocale ITALY = new MapLocale(LOCAL_NAME, ITALY_BBOX);
+  public static final MapLocale JAPAN = new MapLocale(LOCAL_NAME, JAPAN_BBOX);
+  public static final MapLocale KOREA = new MapLocale(LOCAL_NAME, KOREA_BBOX);
+  public static final MapLocale CHINA = new MapLocale(SIMPLIFIED_CHINESE, CHINA_BBOX);
+  public static final MapLocale PRC = new MapLocale(SIMPLIFIED_CHINESE, CHINA_BBOX);
+  public static final MapLocale TAIWAN = new MapLocale(CHINESE, TAIWAN_BBOX);
+  public static final MapLocale UK = new MapLocale(ENGLISH, UK_BBOX);
+  public static final MapLocale US = new MapLocale(ENGLISH, USA_BBOX);
+  public static final MapLocale CANADA = new MapLocale(ENGLISH, CANADA_BBOX);
+  public static final MapLocale CANADA_FRENCH = new MapLocale(FRENCH, CANADA_BBOX);
+
+
+  private static final Map<Locale, MapLocale> LOCALE_SET;
+
+  static {
+    LOCALE_SET = new HashMap<>();
+    LOCALE_SET.put(Locale.US, MapLocale.US);
+    LOCALE_SET.put(Locale.CANADA_FRENCH, MapLocale.CANADA_FRENCH);
+    LOCALE_SET.put(Locale.CANADA, MapLocale.CANADA);
+    LOCALE_SET.put(Locale.CHINA, MapLocale.CHINA);
+    LOCALE_SET.put(Locale.TAIWAN, MapLocale.TAIWAN);
+    LOCALE_SET.put(Locale.PRC, MapLocale.PRC);
+    LOCALE_SET.put(Locale.ITALY, MapLocale.ITALY);
+    LOCALE_SET.put(Locale.UK, MapLocale.UK);
+    LOCALE_SET.put(Locale.JAPAN, MapLocale.JAPAN);
+    LOCALE_SET.put(Locale.KOREA, MapLocale.KOREA);
+    LOCALE_SET.put(Locale.GERMANY, MapLocale.GERMANY);
+    LOCALE_SET.put(Locale.FRANCE, MapLocale.FRANCE);
+  }
```

Comment:
I would favor using double brace initalisation above a static block:
```java
private static final Map<Object, Object> MAP = new HashMap<Object, Object>(){{
    put(new Object(), new Object());
    put(new Object(), new Object());
  }};
```

Reply:
I get this lint warning when moving them into a double brace:

![screen shot 2018-02-21 at 10 16 12 am](https://user-images.githubusercontent.com/5652865/36488008-576e99aa-16f0-11e8-8b83-0db93ef2b216.png)